.github/workflows/update-oam-defs.yml

```
@@ -0,0 +1,119 @@
+name: Linkerd Pattern Components Generator
+on:
+  push:
+    branches:
+      - 'linkerd-comps-test'
```

Hey @DelusionalOptimist just a question might be a silly one but why aren't we using "master" here.😃

---

```suggestion
      - 'master'
```
Sharp eyes. Yes, this is unintended. Thanks @sayantan1413